docs/c-ffi/calling-c.md

```diff
@@ -43,7 +43,11 @@ When dealing with `void*` return types from C, it is good practice to try to nar
 To pass pointers to values to C the `addressof` operator can be used (previously `&`), just like taking an address in C. This is done in the standard library to pass the address of a `U32` to an FFI function that takes a `int*` as an out parameter:
 
 ```pony
---8<-- "calling-c-addressof.pony"
+--8<--
+https://raw.githubusercontent.com/ponylang/ponyc/main/packages/builtin/float.pony:59
```

Comment:
Referencing files in the standard library by line number is rather brittle (if the upstream file changes, the line numbers could silently get out of sync).

We discussed in the sync call today that we'd prefer you to copy these files (or at least some minimal runnable parts of them) into the tutorial repo.

It's not necessary that they exactly match the upstream files or stay in sync with the standard library - all that matters is that the line numbers we highlight here continue to demonstrate the concept.

Reply:
> if the upstream file changes, the line numbers could silently get out of sync

That can happen here just as well, because the relation is one-sided: The snippet doesn't know the place (or places!) where it is used, let alone the relevant lines. Do you have safeguards for that in mind?

> We discussed in the sync call today that we'd prefer you to copy these files (or at least some minimal runnable parts of them) into the tutorial repo.

Duplicate content it is, then. I'll revert the relevant commits 👍🏻